records/tests.py

```diff
@@ -1,29 +1,74 @@
-from django.test import TestCase
+from django.conf import settings
+from django.test import TestCase, override_settings
 from django.contrib.auth import get_user_model
-from rest_framework.test import APIClient
 
 from records.models import Record
 
 
+@override_settings(REDIS_KEY_USER_TAGS='test_tags_%s')
 class RecordsTests(TestCase):
+
     def setUp(self):
+        self._flush_redis()
+        self.user = self._add_user()
+
+    def tearDown(self):
+        self._flush_redis()
+
+    def _flush_redis(self):
+        keys = settings.REDIS_CONN.keys('test_*')
+        if keys:
+            settings.REDIS_CONN.delete(*keys)
+
+    def _add_user(self, username='test'):
         userModel = get_user_model()
-        self.user = userModel(username='test')
-        self.user.save()
+        user = userModel(username=username)
+        user.save()
+        return user
 
-    def _add_record(self):
-        record = Record(user=self.user, transaction_type='EXP', amount=10)
-        record.tags.set_bit(1, True)
-        record.tags.set_bit(5, True)
+    def _add_record(self, user, tags_bits=[]):
+        record = Record(user=user, transaction_type='EXP', amount=10)
+        for bit in tags_bits:
+            record.tags.set_bit(bit, True)
         record.save()
         return record
 
     def test_01_add_record(self):
-        record = self._add_record()
+        record = self._add_record(user=self.user, tags_bits=[1, 5])
```

Comment:
where `1` and `5` comes from? Looks like [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) for me